firmware/common2015/utils/rtos-mgmt/helper-funcs.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+void setISRPriorities(void);
```

Comment:
I haven't really looked into interrupt priorities. I don't expect the base station will need any interrupt priority changes tho.